docs/src/pages/demos/cards/cards.md

```diff
@@ -33,7 +33,7 @@ By default, we use the combination of a `<div>` element and a *background image*
 
 {{"demo": "pages/demos/cards/ImgMediaCard.js"}}
 
-> ⚠️ The previous demo replies on `object-fit` for centering the image. It's not supported by IE 11.
+> ⚠️ In the previous demo, CardMedia relies on `object-fit` for centering the image. It's not supported by IE 11.
```

Comment:
Does mentioning the previous demo add any information here?

Reply:
The limitation only applies when `component="img"` is provided.

Reply:
```suggestion
> ⚠️ When `component="img"`, CardMedia relies on `object-fit` for centering the image. It's not supported by IE 11.
```
?

Reply:
Better! @alltom Can you accept the change?

![Capture d’écran 2019-04-03 à 00 15 18](https://user-images.githubusercontent.com/3165635/55439657-95a39000-55a5-11e9-9f00-7d0923490da4.png)

Reply:
@oliviertassinari How do I accept the change?

Reply:
With the commit suggestion button. But don't worry about it. It's taken into account now.